File: src/main/java/fi/finlit/edith/ui/services/svn/UpdateCallback.java
/*
 * Copyright (c) 2009 Mysema Ltd.
 * All rights reserved.
 * 
 */
package fi.finlit.edith.ui.services.svn;

import java.io.InputStream;
import java.io.OutputStream;

/**
 * UpdateCallback provides
 *
 * @author tiwe
 * @version $Id$
 */
public interface UpdateCallback {
    
    /**
     * @param source
     * @param target
     */
    void update(InputStream source, OutputStream target) throws Exception;

}

File: src/main/scala/fi/finlit/edith/ui/services/svn/UpdateCallback.scala
package fi.finlit.edith.ui.services.svn

import java.io.IOException
import java.io.InputStream
import java.io.OutputStream
//remove if not needed
import scala.collection.JavaConversions._

trait UpdateCallback {

  /**
   * @param source
   * @param target
   */
  def update(source: InputStream, target: OutputStream): Unit
}
